lighthouse-core/config/config-helpers.js

```diff
@@ -6,7 +6,7 @@
 'use strict';
 
 const path = require('path');
-const isDeepEqual = require('lodash.isequal');
+const { isEqual: isDeepEqual } = require('lodash');
```

Comment:
I'd expect this to error because lodash is only commonjs.

How did you verify these changes?

Reply:
I don't think this should error. I didn't do any verification, since I can't seem to run tests locally.

Reply:
Are you using a supported Node version (`>=14.15`)? And I'd expect many tests to fail precisely because of this error. `yarn build-all && yarn unit-cli` should at least display useful messages when failing.

Reply:
On `master` with `node@14.15.1`, `yarn unit-cli` fails with a bunch of these:
```txt
 FAIL  lighthouse-cli/test/cli/cli-flags-test.js
  ● CLI flags › settings are accepted from a file path

    expect(received).toMatchSnapshot()

    Snapshot name: `CLI flags settings are accepted from a file path 1`

    - Snapshot  - 1
    + Received  + 1

    @@ -1,7 +1,7 @@
      Object {
    -   "$0": "node_modules/jest/bin/jest.js",
    +   "$0": "node_modules/jest-worker/build/workers/processChild.js",
        "_": Array [
          "http://www.example.com",
        ],
        "budget-path": "path/to/my/budget-from-config.json",
        "budgetPath": "path/to/my/budget-from-config.json",

      33 |   }
      34 |
    > 35 |   expect(flags).toMatchSnapshot();
         |                 ^
      36 | }
      37 |
      38 | describe('CLI flags', function() {

      at snapshot (lighthouse-cli/test/cli/cli-flags-test.js:35:17)
      at Object.<anonymous> (lighthouse-cli/test/cli/cli-flags-test.js:76:5)
```